### PR TITLE
🔧 buy now stroke cart icon

### LINKED ISSUE #6655: Cart svg icon
Looks like it's not coming from the same icon pack

Screenshot 2023-08-10 at 16-02-51 KodaDot - NFT Market Explorer

What can we improve:
- fix size
- fix line weight
- replace icon with [FontAwesome equivalent](https://fontawesome.com/search?q=cart&o=r)

wdyt @exezbcz

### PR DESCRIPTION
**Pull request by roiLeo**
- close #6655
- https://github.com/kodadot/nft-gallery/issues/6655#issuecomment-1793844214

Capture d’écran 2023-11-16 à 3 59 49 PM

A quick and ugly solution, but at what cost?

### CODE REVIEW

**Comment by prury** [comment]
for me, previously was not working only on Asset hubs, fixed now!

File: components/items/ItemsGrid/ItemsGridImageTokenEntity.vue [hunk 1]
```diff
@@ -156,7 +163,8 @@ const listLabel = computed(() => {
   return isInCart ? label + ' ✓' : label
 })
 
-const { cartIcon } = useShoppingCartIcon(props.entity.id)
+// const { cartIcon } = useShoppingCartIcon(props.entity.id)
```
Comment (Jarsen136) [inline components/items/ItemsGrid/ItemsGridImageTokenEntity.vue:166]:
```suggestion
```

**Review by Jarsen136 — APPROVED** [review]

**Comment by exezbcz** [comment]
was the icon added to our fontawesome account or? That was the main goal afaik. I could not log in.

**Comment by roiLeo** [comment]
This PR doesnt close original issue.
Ping @preschian maybe know how to import icôn Into FA pro.

**Comment by preschian** [comment]
@roiLeo @exezbcz uploaded

- `<i class="fa-kit fa-shopping-cart-outline-sharp"></i>`
- `<i class="fa-kit fa-striked-out-cart-shopping"></i>`

**Comment by roiLeo** [comment]
✅ updated

Capture d’écran 2023-11-17 à 4 45 19 PM

Screenshot 2023-11-17 at 16-44-52 Cy3
Screenshot 2023-11-17 at 16-35-58 KodaDot - NFT Market Explorer

**Comment by exezbcz** [comment]
@roiLeo nice! could you please make the icon in the navbar 24*24px 
image

- I will leave the rest for followup

**Comment by roiLeo** [comment]
> @roiLeo nice! could you please make the icon in the navbar 24*24px image
> 
>     * I will leave the rest for followup

sure, note that now it won't have the same size as other

Screenshot 2023-11-17 at 17-06-29 KodaDot - NFT Market Explorer
Capture d’écran 2023-11-17 à 5 09 06 PM

**Comment by exezbcz** [comment]
> edit: made a change to unify icon size in navbar

thank you sir, wfm

**Comment by prury** [comment]
working for me as well

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Pull request merged**